package.json

```diff
@@ -1,41 +1,56 @@
 {
   "name": "bless",
-  "description": "CSS Post-Processor",
+  "description": "CSS Post-Processor that chunks CSS files per IE9 selector count restrictions",
   "url": "http://blesscss.com",
   "keywords": [
     "css",
     "parser",
-    "bless",
     "less",
     "sass",
-    "stylus"
+    "stylus",
+    "ie9",
+    "selector-limit"
   ],
   "author": "Paul Young <paulyoungonline+bless@gmail.com>",
-  "version": "4.0.0",
+  "maintainers": [
+    "Matt Smith <mtscout6+bless@gmail.com>",
+    "Alex Abenoja <abenojaalex@gmail.com>"
+  ],
+  "repository": "BlessCSS/bless",
+  "version": "4.2.4",
```

Comment:
Can we hold off on incrementing this, or at least use something like `4.0.0-alpha`?